libhb/muxavformat.c

```diff
@@ -197,6 +197,11 @@ static int avformatInit( hb_mux_object_t * m )
                      &m->oc->interrupt_callback, NULL);
     if( ret < 0 )
     {
+      if( ret == -2 ) 
+      {
+        hb_error( "avio_open2 failed, errno -2: Could not write to indicated output file." );
```

Comment:
Maybe worth adding "Please check destination and file permissions"

Reply:
I added an extra sentence to include this